pywps/exceptions.py

```diff
@@ -34,7 +34,7 @@ class NoApplicableCode(HTTPException):
     Base exception class
     """
 
-    code = 400
+    code = 500
     locator = ""
 
     def __init__(self, description, locator="", code=400):
```

Comment:
hmm makes sense

btw, just refer to https://en.wikipedia.org/wiki/List_of_HTTP_status_codes we should use them more